core/templates/dev/head/pages/exploration_editor/add_exploration_metadata_modal.html

```diff
@@ -0,0 +1,58 @@
+<script type="text/ng-template" id="modals/addExplorationMetadata">
+      <div class="modal-header">
```

Comment:
The indentation here is wrong; it should be 2 more than that of the previous line. Ditto through the rest of the file.